styles/styles.css

```diff
@@ -369,6 +806,56 @@
 
     /* Block specific */
     --signup-padding: var(--spacing-52);
+
+    /* numbers */
+    --border-radius-none: 0rem;
+    --border-radius-small: 0.125rem;
+    --border-radius-base: 0.25rem;
+    --border-radius-medium: 0.5rem;
+    --border-radius-large: 1rem;
+    --border-radius-x-large: 2rem;
+    --global-gutter: 1.5rem;
```

Comment:
From my understanding, we have defined all the values as variables at the top of the code. Shouldn't we follow the same approach here? Additionally, since the rem values are frequently used in the variables, defining them as separate variables would simplify maintenance if updates are needed in the future. Just a thought.